src/einsteinpy/symbolic/predefined/find.py

```diff
@@ -0,0 +1,11 @@
+import importlib
+
+
+def find(search_string):
+    current_module = "src.einsteinpy.symbolic.predefined"
+    predefined = importlib.import_module(current_module)
+    return [
+        fname
+        for fname in dir(predefined)
+        if search_string in fname.lower() and not fname.islower()
```

Comment:
```suggestion
        if ((search_string in fname.lower()) and (not fname.islower()))
```

a few brackets would improve clarity :)